ur_bringup/launch/ur_moveit.launch.py

```diff
@@ -355,5 +369,8 @@ def generate_launch_description():
     declared_arguments.append(
         DeclareLaunchArgument("launch_rviz", default_value="true", description="Launch RViz?")
     )
+    declared_arguments.append(
+        DeclareLaunchArgument("launch_servo", default_value="true", description="Launch Servo?")
```

Comment:
I made it so Servo is launched by default. If the user doesn't need it, it sits quietly, waiting for a service call to activate.